### PR TITLE
PB-1856 Read the values from kdmp-config configmap instead of env

### PR DESCRIPTION
**Pull request by prabhu-px**
PB-1856 Read the values from kdmp-config configmap instead of env
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** (optional)
Closes # PB-1856

**Special notes for your reviewer**:

### CODE REVIEW

File: pkg/drivers/utils/utils.go [hunk 1]
```diff
@@ -73,9 +74,22 @@ func ToJobStatus(progress float64, errMsg string) *drivers.JobStatus {
 	}
 }
 
+// GetConfigValue read configmap and return the value of the requested parameter
+func GetConfigValue(key string) string {
+	configMap, err := core.Instance().GetConfigMap(
+		"kdmp-config",
+		"kube-system",
+	)
+	if err != nil {
```
Comment (ram-infrac) [inline pkg/drivers/utils/utils.go:83]: it should be - 
```
	if err != nil {
			log.Errorf("Failed to read configmap. :%v",err)
	                return ""
	}
        return configMap.Data[key]
}
```

File: pkg/drivers/utils/utils.go [hunk 2]
```diff
@@ -73,9 +74,22 @@ func ToJobStatus(progress float64, errMsg string) *drivers.JobStatus {
 	}
 }
 
+// GetConfigValue read configmap and return the value of the requested parameter
+func GetConfigValue(key string) string {
+	configMap, err := core.Instance().GetConfigMap(
+		"kdmp-config",
```
Comment (ram-infrac) [inline pkg/drivers/utils/utils.go:80]: make this constant

**Review by ram-infrac — APPROVED** [review]
lgtm after fixing comments

**Review by prashanthpx — APPROVED** [review]
looks good to me (no other comments apart from the current ones)

File: pkg/drivers/utils/utils.go [hunk 3]
```diff
@@ -73,9 +78,22 @@ func ToJobStatus(progress float64, errMsg string) *drivers.JobStatus {
...
+func GetConfigValue(key string) string {
+	configMap, err := core.Instance().GetConfigMap(
+		kdmpConfig,
+		defaultPXNamespace,
+	)
+	if err != nil {
+		return ""
+	}
+	log.Errorf("Failed to read configmap.")
+	return configMap.Data[key]
```
Comment (adityadani) [inline pkg/drivers/utils/utils.go:91]: If a config map is not found, can we check if there is an env variable?

File: pkg/drivers/utils/utils.go (hunk 1 above)
Comment (siva-portworx) [reply pkg/drivers/utils/utils.go:92]: Fixed

File: pkg/drivers/utils/utils.go (hunk 3 above)
Comment (siva-portworx) [reply pkg/drivers/utils/utils.go:97]: Fixed

File: pkg/drivers/utils/utils.go (hunk 2 above)
Comment (siva-portworx) [reply pkg/drivers/utils/utils.go]: Fixed

**Review by prashanthpx — APPROVED** [review]
lgtm

**Pull request merged**